setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='csv_template_ingester',
-    version='1.1.0',
+    version='1.1.1',
```

Comment:
This is a new feature, so it should be a minor version bump.  See [semver](https://semver.org/)

Reply:
rgr, I'll get on this